e2e/step_definitions/programCatalog.js

```diff
@@ -18,11 +25,7 @@ defineSupportCode(({Given, When, Then, setDefaultTimeout}) => {
         return expect(ProgramCatalog.isSearchBarVisible()).to.eventually.equal(visibility === 'visible');
     });
 
-   /*Then(/^the placeholder should be visible$/, () => {
-
-    });*/
-
-    Then(/^the placeholder text should be "(.*)"$/, function (text) {
+    Then(/^the placeholder text should be "(.*)"$/, text => {
```

Comment:
Please, modify the regex so that it works if there is a " character in the search term.
([^"].*)